### PR TITLE
Basic web UI built with Flask

### PR DESCRIPTION
**Pull request by dsheehan**
Frontend UI built with Flask.
Use requirements.txt for venv/pip.
Dockerfile now bundles entire /app/ directory.
Pull version from config.json.
Replace print() statements with log.info()

### CODE REVIEW

**Comment by mrlt8** [comment]
Thank you, this is awesome! This has been requested numerous times, so I'm sure many users will be excited as well!

I was even able to get it to load in home assistant, but we may need to tweak the domain for the hrefs  as I believe some people use a custom ddns.

**Comment by dsheehan** [comment]
@mrlt8 there are env vars to support custom dns domains, which will support various configurations like different ports, subdomains or reverse proxies in front.

WB_HLS_URL, WB_RTMP_URL, WB_RTSP_URL

```
        self.wb_hls_url = os.getenv("WB_HLS_URL", f"http://{localhost}:8888/")
        self.wb_rtmp_url = os.getenv("WB_RTMP_URL", f"rtmp://{localhost}:1935/")
        self.wb_rtsp_url = os.getenv("WB_RTSP_URL", f"rtsp://{localhost}:8554/")
```

File: app/wyze_bridge.py [hunk 1]
```diff
@@ -434,25 +435,29 @@ def get_webrtc(self):
...
+            d["rtmp_url"] = self.wb_rtmp_url + cam.name_uri
+            d["rtsp_url"] = self.wb_rtsp_url + cam.name_uri
+            d["name_uri"] = cam.name_uri
+            d["connected"] = (
+                self.streams[cam.nickname]["connected"].is_set()
+                if "connected" in self.streams[cam.nickname]
+                else False
+            )
+            if img := os.path.exists(f"{self.img_path}{cam.name_uri}.jpg"):
+                d["img"] = f"/local/{cam.name_uri}.jpg" if self.hass else img
```
Comment (dsheehan) [inline app/wyze_bridge.py:460]: If it's HASS shouldn't {self.img_path} have already been set correctly upstream, it should be safe to prefix with img_path in either mode.

**Comment by mrlt8** [comment]
@dsheehan any idea what is causing this?
hls

**Comment by dsheehan** [comment]
@mrlt8 I will remove the hls.js video stuff from this PR, and we can figure out the live HLS <video> element feeds in a subsequent PR.

**Pull request merged**